include/inviwo/core/ports/port.h

```diff
@@ -59,7 +59,7 @@ class IVW_CORE_API Port : public Serializable {
     Processor* getProcessor() const;
 
     /**
-     * @brief Get the port path i.e. '<processor identifier>.<port identifier>'
+     * @brief Get the port path i.e. ```<processor identifier>.<port identifier>```
```

Comment:
sure about the triple "`"? Isn't that a block?

Reply:
seems to work http://jenkins.inviwo.org:8080/job/inviwo/job/PR-1080/Doxygen/classinviwo_1_1Port.html#a013b5886880e37e693fe988ed10f5348